core/src/test/java/greencity/controller/CategoryControllerTest.java

```diff
@@ -1,61 +1,86 @@
 package greencity.controller;
 
+
 import greencity.dto.category.CategoryDto;
 import greencity.service.CategoryService;
+
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+
 import org.springframework.data.web.PageableHandlerMethodArgumentResolver;
 import org.springframework.http.MediaType;
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.setup.MockMvcBuilders;
 
-import static org.mockito.Mockito.times;
+
 import static org.mockito.Mockito.verify;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
+
+
 @ExtendWith(MockitoExtension.class)
 class CategoryControllerTest {
 
-    private MockMvc mockMvc;
-
     @Mock
-    CategoryService categoryService;
+    private CategoryService categoryService;
 
     @InjectMocks
     CategoryController categoryController;
 
     private static final String categoryLink = "/category";
 
+    private MockMvc mockMvc;
+
+
     @BeforeEach
     void setup() {
         this.mockMvc = MockMvcBuilders.standaloneSetup(categoryController)
-            .setCustomArgumentResolvers(new PageableHandlerMethodArgumentResolver())
-            .build();
+                .setCustomArgumentResolvers(new PageableHandlerMethodArgumentResolver())
+                .build();
     }
 
     @Test
+    @DisplayName("Category Controller Post request to create new Category Test")
     void saveCategory() throws Exception {
-        CategoryDto categoryDto = CategoryDto.builder().name("content").build();
+        String name = "Category Name";
+        String nameUa = "Категорія";
+        long id = 5;
+
+        CategoryDto categoryDto = CategoryDto.builder()
+                .name(name)
+                //.nameUa(nameUa)
+                //.parentCategoryId(id)
+                .build();
+
+        String json = "{\n" +
+                "  \"name\": \""+ name +"\"\n" +
```

Comment:
better avoid commets

Reply:
there is a bug with Category DTO mapper that's why these 2 line with setting nameUa & id are not working. So I've commented it to uncomment when bug is fixed

Reply:
removed it for now